index.js

```diff
@@ -95,8 +96,16 @@ Wemo.prototype._handleRequest = function(req, res) {
 };
 
 Wemo.prototype.getCallbackURL = function() {
+  var self = this;
   var getLocalInterfaceAddress = function() {
     var interfaces = os.networkInterfaces();
+    if (self._callbackInterface) {
```

Comment:
Sure I can do that.  Since the new option specifies the name of the interface and not the address, should the server listen on the IPv4 address for the given interface name?